simple_certmanager/utils.py

```diff
@@ -19,8 +19,20 @@ def load_pem_x509_private_key(data: bytes):
     return load_pem_private_key(data, password=None)
 
 
+def _decode(value: Union[str, bytes]) -> str:
+    # attr.value can be bytes, in which case it is must be an UTF8String or
+    # PrintableString (the latter being a subset of ASCII, thus also a subset of UTF8)
+    # See https://www.rfc-editor.org/rfc/rfc5280.txt
+    if not isinstance(value, bytes):
+        # https://github.com/microsoft/pyright/issues/6059 Pyright does not like this :(
+        return value
+    return value.decode("utf8")
```

Comment:
Why not `django.utils.text.smart/force_str`?

Reply:
Yeah I wasn't sure if it was appropriate because those utils aren't limited to `str` or `bytes`, but we should only get those types so I've updated it.